### PR TITLE
set loglevel as opt

### PR DESCRIPTION
**Pull request by mbussolotto**
Since urlgrabber can also be used as a library, we would like to be able to set log level as an option, without creating the URLGRABBER_DEBUG env variable

Fixes https://github.com/SUSE/spacewalk/issues/18918
Issue https://github.com/uyuni-project/uyuni/issues/6099

### CODE REVIEW

**Comment by mbussolotto** [comment]
cc @meaksh

File: urlgrabber/grabber.py
Comment (meaksh) [inline urlgrabber/grabber.py:1228]: I would consider adding this also to the other public methods of `URLGrabber` class: `urlopen` and `urlread`.

**Review by meaksh — APPROVED** [review]
Looks good to me :+1:

**Comment by mbussolotto** [comment]
@meaksh I just realized I cannot merge the PR cause  don't have write acess to the repo...can you please merge it (or provide me write access, as you prefer)? Thanks!

**Comment by mbussolotto** [comment]
cc @vzhestkov

File: urlgrabber/grabber.py
Comment (vzhestkov) [inline urlgrabber/grabber.py:663]: Maybe better this way?
```suggestion
        if isinstance(logspec, list):
            dbinfo = logspec
        elif "," in logspec:
            dbinfo = logspec.split(",")
```

Comment (mbussolotto) [reply urlgrabber/grabber.py:663]: agree, same result but more clean. Thanks for the suggestion

**Review by vzhestkov — APPROVED** [review]
Looks good to me :+1:

**Review by Conan-Kudo — APPROVED** [review]

**Comment by Conan-Kudo** [comment]
> Fixes https://github.com/SUSE/spacewalk/issues/18918

In the future, please don't reference things nobody can look at. It's rude and makes people think there's missing context. I merged it anyway because the change is simple enough that it's pretty standalone, but keep this in the mind going forward.

**Pull request merged**
